src/main/java/com/airlines/model/Crew.java

```diff
@@ -1,16 +1,14 @@
 package com.airlines.model;
 
 import java.util.List;
+import java.util.Objects;
 
 public class Crew {
-    private Long id;
-    private String name;
-    private List<CrewMember> crewMembers;
+    private final Long id;
+    private final String name;
+    private final List<CrewMember> crewMembers;
 
-    public Crew() {
-    }
-
-    public Crew(Long id, String name, List<CrewMember> crewMembers) {
+    Crew(Long id, String name, List<CrewMember> crewMembers) {
         this.id = id;
         this.name = name;
         this.crewMembers = crewMembers;
```

Comment:
id and name are immutable, we can't change them after Crew instance is created.
But List<CrewMember> could be changed, because final modifier means we are obligated to assign reference to some List object to our variable - clas field crewMembers, and we can not reattach other list.

But we can change state of this list:
Crew crew = new Crew(1L, "Blue Team", crewMembers);
List<CrewMember> list = cerw.getCrewMembers();
list.clear();
list.add(badCrewMember);

and so on.

So as long as we can get link to mutable object, we can change state, so Crew is mutable, we can change it any time.

I don't know if we need immutable there, but to get known with this moment I suggest making field "crewMembers" immutable

you can do it in constructor.

Google
"how make list immutable java" there special Collections class method